coastlines/raster.py

```diff
@@ -510,6 +498,8 @@ def generate_rasters(
     end_year,
     tide_centre,
     buffer,
+    tide_model="EOT20",
```

Comment:
To allow us to customise tide model selection for DEA Coastlines (which is a first! It's been hard-coded previously)